### DISCUSSION TITLE
Gettings the pre-releases out of the release folder - Possibility to ignore pre-releases in the notifications?

### DISCUSSION THREAD

**Discussion by TMillesich** [podman-desktop/podman-desktop#7028] [General]
Hello,
as much as i like it to see how active the development is - is there a way to get the pre-releases out of the release folder, or at least don't get any notifications for the pre-releases?

I've more than 100 emails with "[containers/podman-desktop] Pre-release v0.0.2" in my inbox.

Sadly it looks like a common GitHUB problem - i could not find any solution. 🤔

**Comment by afbjorklund** [comment]
Podman Desktop is doing a new (pre)release for every commit, which breaks the system that is only intended for releases

* https://github.com/containers/podman-desktop/issues/6758

Compare with Podman, where you have your pre-releases ("Release Candidates" / public betas) and releases (remote-only)

* https://github.com/containers/podman/releases
